### PR TITLE
Use NSIS to download and extract bootstraps

### PR DESCRIPTION
**Pull request by Luke-Larsen**
This pull requires the NSIS plugin https://github.com/negrutiu/nsis-nscurl to be installed.

NSIS curl will pull the bootstrap down then we are using powershell to extract the bootstrap to the right place. Due to the sizes of the bootstrap file (More then 2 GB) all the tested NSIS plugins will not work.

### CODE REVIEW

File: share/setup.nsi.in [hunk 1]
```diff
@@ -154,6 +154,19 @@ no_smgroup:
     Pop $R0
 SectionEnd
 
+Section "Bootstrap" SecBootstrap
+    # Download for our bootstrap
+    NScurl::http GET "https://explorer.decenomy.net/bootstraps/SAPP/bootstrap.zip" "$APPDATA\__Decenomy__\bootstrap.zip" /POPUP /END
```
Comment (crypt0p3t3r) [inline share/setup.nsi.in]: where there is SAPP it should be __DSW__

Comment (bedri) [reply share/setup.nsi.in]: @Luke-Larsen if you wont update this pull request and fix what @pedro-at-decenomy mentioned, I will open a new pull request with the correction pointed out by Pedro.
Are you still interested in this pull request and will you update it Luke?

**Comment by bedri** [comment]
Is this pull request checked @pedro-at-decenomy ?
If checked and ok, can we merge or close this pull request please?
It had been waiting for some long time :)

**Comment by crypt0p3t3r** [comment]
The work will be continued on another PR

**Pull request closed without merging**